spec/features/admin_edits_piece_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+feature "admin edits pieces" do
+  context "visiting piece show page" do
+    before do
+      dumpling_king = Piece.create(title: 'Dumpling King', composer: "Tchackowskowitz")
+
+      visit piece_path(dumpling_king)
+      click_link "Edit Piece"
+    end
+
+    scenario "admin sees edit form for piece" do
+
+      expect(page).to have_content "Edit Piece"
+
+      title_and_composer
+      click_button "Edit"
+
+      expect(page).to have_content "Marriage of Figaro"
+      expect(page).to have_content "Mozart"
+    end
```

Comment:
I see the "user" clicks the edit button, but they don't change any of the fields and click "Update Piece".